### PR TITLE
dynamically add healthkit entitlements

### PR DESCRIPTION
**Pull request by johnborges**
As of cordova-ios@4.3.0 you can make direct use of a debug and release entitlement plist to dynamically add entitlements. No need for hooks. This plugin can directly add the entitlements it needs. https://github.com/Telerik-Verified-Plugins/HealthKit/issues/79

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
   <engines>
     <engine name="cordova" version=">=3.0.0"/>
+    <engine name="cordova-ios" version=">=4.3.0" />
```
Comment (EddyVerbruggen) [inline plugin.xml]: I ❤️  that we can do this, but can we remove this restriction to not impose 4.3.0+ on people? I guess those `config-file` lines will just be ignored on lower versions and we can update the readme to state manually adding the entitlements is only required for < 4.3.0.

WDYT?

Thanks,
Eddy

Comment (johnborges) [inline plugin.xml]: I think that is a good idea as long as those `config-file` lines get ignored gracefully. I'll try and test it out `cordova-ios` < 4.3.0

Comment (EddyVerbruggen) [inline plugin.xml]: Would be awesome if that's the case!

**Comment by johnborges** [comment]
So it looks like on older projects which make use of `cordova-ios<4.3.0` the plugin works fine and simply ignores the references made to `Entitlements-Debug` and `Entitlements-Release`. No need to enforce `cordova-ios@4.3.0` requirement.

**Comment by EddyVerbruggen** [comment]
That's perfect! I guess we're ready to merge then, right?

**Comment by johnborges** [comment]
yup

**Comment by EddyVerbruggen** [comment]
Awesomness, will release to npm soon 🎉

**Pull request merged**